app/presenters/schools/courses/curriculum_presenter.rb

```diff
@@ -15,7 +15,8 @@ def props
           evaluationCriteria: evaluation_criteria,
           levels: levels,
           targetGroups: target_groups,
-          targets: targets
+          targets: targets,
+          vimeoAccessToken: vimeo_access_token
```

Comment:
The method name is missing the trailing question mark.

```suggestion
          vimeoAccessToken: vimeo_access_token?
```